### ISSUE TITLE
Not linting

### ISSUE THREAD

**Issue by svbeon** [ricardofbarros/linter-js-standard#222]
Hi,

Since a few weeks back the linter has just stopped linting, without errors or anything.
I'm using `busy-signal (1.4.3)`, `intentions (1.1.5)` , `linter (2.2.0)`, `linter-ui-default (1.7.1)`, `linter-js-standard (5.1.0)` on `atom (1.30.0)` 

I realize that this might be an issue with Linter, if so please tell and I'll move this issue. Do anyone have a solution or at least a way for me to debug this better?

The Linter UI displays fine but remains empty, I've tried disabling all other packages without luck, I've tried removing and reinstalling the packages and resetting all their settings to default. Running the `Linter: Lint` command gives no console output, nothing at all happens. running `Linter: Debug`  yields

```
Platform: win32
Atom Version: 1.30.0
Linter Version: 2.2.0
Opened file is ignored: No
Matching Linter Providers: 
  - js-standard
Disabled Linter Providers: 
Standard Linter Providers: 
  - js-standard
Indie Linter Providers: 
UI Providers: 
  - Linter
Ignore Glob: **/*.min.{js,css}
VCS Ignored Paths are excluded: true
Current File Scopes: 
  - *
  - source.js
  - meta.function-call.js
  - meta.arguments.js
  - meta.brace.round.js
```

Linter is set as follows:
  - [x] Lint preview tabs
  - [x] Lint on open
  - [x] Lint on change
  - Lint on change interval: `300` (default)
  - Ignore files matching this glob: `**/*.min.{js,css}` (default)
  - Disabled providers: `` (default)

linter-js standard is set as follows:
  - [ ] Disable if the project uses ESLint
  - [ ] Check for standard
  - [ ] Honor style settings
  - [ ] Lint HTML Files
  - [ ] Lint Markdown Files
  - [ ] Show ESLint rules
  - Style: `standard`

linter-ui-default is set as follows:
  - [ ] Always take minimum space
  - [x] Hide panel when empty
  - [x] Show decorations
  - [x] Show panel
  - [ ] Show provider name
  - [x] Show status bar
  - [x] Show tooltip
  - [x] Use busy signal
  - Decorate on Tree View: `Files`
  - Gutter Highlights position: `High` 
  - Panel Represents: `Current file`
  - Statusbar click behaviour: `Toggle panel`
  - Statusbar position: `Left`
  - Statusbar represents: `Entire Project`
  - Tooltip follows: `Both`
  - Panel Height: `162`

**Comment by sonicdoe** [comment]
Thanks for the detailed description but I couldn’t reproduce this on my Windows machine. Does `standard` on the command line lint correctly?

**Comment by svbeon** [comment]
@sonicdoe yup, no wait in one project folder node has f-ed up the permissions on a log file... that might be it... however it seems odd to me that that error isn't communicated to the user either in the linter-ui or in the devtools console

```
standard: Unexpected linter output:

Error: EPERM: operation not permitted, scandir 'C:\Hemarbete\.babun\cygwin\home\00svbeon\git\kulla\firebase-debug.log'

If you think this is a bug in `standard`, open an issue: https://github.com/feross/standard/issues
glob error { Error: EPERM: operation not permitted, scandir 'C:\Hemarbete\.babun\cygwin\home\uname\git\project\npm-package.log'
  errno: -4048,
  code: 'EPERM',
  syscall: 'scandir',
  path: 'C:\\Hemarbete\\.babun\\cygwin\\home\\uname\\git\\project\\npm-package.log' }
``

**Comment by svbeon** [comment]
Sorry for not reporting back for a few days, i tried removing that project folder from atom, restarted and it still own't work.

I have gotten eslint to run now so in the meantime i have installed standards eslint rules

**Comment by sonicdoe** [comment]
I was able to reproduce your issue when the project folder is within a hidden folder (as it is the case with your `C:\Hemarbete\.babun` path). I don’t have a fix ready yet but will try to get to it soon.

**Comment by sonicdoe** [comment]
This should now be fixed in linter-js-standard v6.0.0. Please let me know if you are still running into issues.

**Comment by svbeon** [comment]
Thanks, will give it a try later today
